README.md

```diff
@@ -386,9 +386,9 @@ These require you to set `PICO_ARM_TOOLCHAIN_PATH` and `PICO_RISCV_TOOLCHAIN_PAT
 
 App|Description
 ---|---
-[blink](universal/CMakeLists.txt#L126) | Same as the [blink](blink) example, but universal.
+[blink](universal/CMakeLists.txt#L137) | Same as the [blink](blink) example, but universal.
```

Comment:
Why are the line numbers there at all?

Reply:
According to git blame, the line-numbers first appeared in https://github.com/raspberrypi/pico-examples/commit/7fe60d6b4027771e45d97f207532c41b1d8c5418
Ping @will-v-pi and @kilograham

Reply:
It’s because those examples don’t have their own folders, they’re just created by lines in the CMakeLists.txt file. The actual example code is in the normal blink/nuke folder.